src/Illuminate/Support/Testing/Fakes/HashFake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Illuminate\Support\Testing\Fakes;
+
+use Illuminate\Contracts\Hashing\Hasher;
+
+class HashFake implements Hasher
+{
+    /**
+     * Get information about the given hashed value.
+     *
+     * @param  string  $hashedValue
+     * @return array
+     */
+    public function info($hashedValue)
+    {
+        return password_get_info($hashedValue);
+    }
+
+    /**
+     * Hash the given value.
+     *
+     * @param  string  $value
+     * @param  array   $options
+     * @return string
+     */
+    public function make($value, array $options = [])
+    {
+        return crc32($value);
+    }
+
+    /**
+     * Check the given plain value against a hash.
+     *
+     * @param  string  $value
+     * @param  string  $hashedValue
+     * @param  array   $options
+     * @return bool
+     */
+    public function check($value, $hashedValue, array $options = [])
+    {
+        return crc32($value) == $hashedValue;
```

Comment:
before anyone asks - this is intentional to be a non-strict check. `crc32()` returns `int` - while `$hasedValue` is likely to be a string.